change/@ni-nimble-components-95871031-15d3-48ad-952f-7c70831fb3d0.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Even though this is a breaking change, since the component is still marked as "in development" we can call it a minor change to signal to our clients that nothing is breaking that they're actually using.

```suggestion
  "type": "patch",
```